ci/kind/test-e2e-kind.sh

```diff
@@ -42,8 +42,8 @@ FLOW_VISIBILITY_CMD=$(dirname $0)"/../../hack/generate-manifest.sh"
 CH_OPERATOR_YML=$(dirname $0)"/../../build/charts/theia/crds/clickhouse-operator-install-bundle.yaml"
 SPARK_OPERATOR_YML=$(dirname $0)"/../../build/yamls/spark-operator.yaml"
```

Comment:
PR #31 is adding the spark operator in build/charts/theia - perhaps we only need it in one place.

@yanjunz97 do you think it might help if we rebase this PR to include your commit from PR #31?

Reply:
Rebase this PR on PR #31 will introduce more changes. PR #31 support to generate a `flow-visibility.yaml` with Spark Operator enabled by running `generate-manifest.sh --spark-operator`. And we do not need a separate `spark-operator.yaml`.